pages/report-hazard/script.js

```diff
@@ -1001,3 +1002,12 @@ reportHazardForm.addEventListener('scroll', function (event) {
     continueBtnContainer.style.boxShadow = 'none';
   }
 });
+
+//imprimr currentReport siempre
```

Comment:
Please stick to only English

Reply:
Sorry that is just for testing purposes I should have deleted everything after line 1005